Framework/DataHandling/test/LoadILLSANSTest.h

```diff
@@ -467,6 +467,26 @@ class LoadILLSANSTest : public CxxTest::TestSuite {
     checkWavelength(outputWS, 4.8);
   }
 
+  void test_D16_OMEGA_NEW() {
```

Comment:
Could we rename the test to e.g. `test_D16B_OMEGA` just in case the new stops being so new in the future?